skbeam/io/tests/test_fit2d_save.py

```diff
@@ -44,16 +44,18 @@
 import numpy as np
 from skbeam.io.fit2d import fit2d_save, read_fit2d_msk
 from numpy.testing import assert_array_equal
+from tempfile import tempdir
 
 
 def test_save_output_fit2d():
+    tdir = tempdir()
```

Comment:
That's neat, but I don't think it's worth introducing a new dependency to solve this problem. Python 2 does provide [this tool for creating a temporary directory](https://docs.python.org/2/library/tempfile.html#tempfile.mkstemp); it's just not as nice as the one in Python 3 because you have to do the cleanup manually. (I suggest `shutil.rmtree`.)